packages/tizen_audio_manager/tizen/src/audio_manager.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_PLUGIN_AUDIO_MANAGER_H_
+#define FLUTTER_PLUGIN_AUDIO_MANAGER_H_
+
+#include <sound_manager.h>
+
+#include <functional>
+#include <string>
+
+using VolumeChangedListener =
+    std::function<void(std::string type_str, int32_t volume)>;
+
+class AudioManager {
+ public:
+  AudioManager() {}
+  ~AudioManager() {}
+
+  std::string getCurrentPlaybackType();
+
+  int32_t getMaxLevel(std::string getMaxLevel);
+
+  void SetLevel(std::string type, int32_t volume);
```

Comment:
nit: volume vs. level vs. volume level?

We should use consistent wording as much as possible.

FYI) This plugin has been named after the C# class [AudioManager](https://samsung.github.io/TizenFX/stable/api/Tizen.Multimedia.AudioManager.html) (and [AudioVolume](https://samsung.github.io/TizenFX/stable/api/Tizen.Multimedia.AudioVolume.html)) but this C++ `AudioManager` class is actually a wrapper around the native Sound Manager module. "volume" may be preferred over "level" if you want to follow the native module's naming convention. However please note that this has nothing to do with the names in the `AudioManagerTizenPlugin` class.

Reply:
done